doc/filters.texi

```diff
@@ -10356,6 +10356,11 @@ The x and y offset coordinates where the box is drawn.
 @item h
 The width and height of the drawn box.
 
+@item box_source
+The box source of drawn box, which can be side_data_detection_bbox. So it will use box infomation in
+detecton bbox of side data, which includes @var{x}, @var{y} offset and @var{width}, @var{height}
```

Comment:
which includes x, y, w and h ...

If we use @var{x} ..., it confuses me with the next line of @var{x}.  What really happens to it?

Reply:
What i want to say is that: if the box source is set mannually, the previous x, y, width and height will be overwritten. Which means, new x, y, width and height of box value will be assigned to variables: x, y, width and height.

Reply:
sure, i understood it. but it will make others confused.  it is better to describe as for vf_drawtext.